pkg/utils/resources/suite_test.go

```diff
@@ -0,0 +1,392 @@
+/*
```

Comment:
Should we have a test that validates our `MergeRequestsWithLimits` logic? Right now we don't have a test that represents the case where we don't have requests but we get limits. In this case, the apiserver represents the requests as the limits